arlo-client/src/components/CreateAudit.tsx

```diff
@@ -79,6 +79,11 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       toast.error(err.message)
     }
   }
+
+  if (isAuthenticated === null) return null // Still loading
```

Comment:
I know this `isAuthenticated` variable isn't introduced in this PR, but it's a bit fragile to treat a boolean as a trinary value where `false` and `null` aren't the same thing. We should probably fix this at some point.

Reply:
good call. i actually think the variable is unnecessary. i explored refactoring a bit to having a user type that included anonymous users, and then just returning the user from this hook (`null` if still loading). but i agree it's not a priority